### DISCUSSION TITLE
Not able to import Data through API's

### DISCUSSION THREAD

**Discussion by VivekGupta97** [apache/superset#18734] [General]
Hi everyone , Actually I want to ask about the import API of superset .
As we know the API documentation is not quite understandable of superset. I was able to export data through API's successfully but when I tried to import data to the superset sometimes it shows Internal server error and sometimes it shows user conformation needed.
If anyone could help me to resolve this issue it will be really appreciable

**Comment by VivekGupta97** [comment]
![Screenshot (103)](https://user-images.githubusercontent.com/68772150/154018938-6127b459-a424-4999-bba5-edbe2aeeb961.png)

**Reply by zhaoyongjie** [reply]
@VivekGupta97 could you detail 
1. post url
2. post header
3. post body

error post response as well.

**Reply by VivekGupta97** [reply]
Status is 200 i.e. ok But when I preview the response then it shows user conformation needed

**Comment by VivekGupta97** [comment]
`{
  "active": true,
  "chart": 28,
  "context_markdown": "string",
  "creation_method": "string",
  "crontab": "* * * * *",
  "dashboard": 0,
  "database": 0,
  "description": "Daily sales dashboard to marketing",
  "grace_period": 14400,
  "log_retention": 90,
  "name": "Daily dashboard email",
  "owners": [
    0
  ],
  "recipients": [
    {
      "recipient_config_json": {
        "target": "string"
      },
      "type": "Email"
    }
  ],
  "report_format": "PNG",
  "sql": "SELECT value FROM Count of ID_Objects",
  "timezone": "string",
  "type": "Alert",
  "validator_config_json": {
    "op": "<",
    "threshold": 0
  },
  "validator_type": "not null",
  "working_timeout": 3600
}`
This Code in Body Tag

**Comment by VivekGupta97** [comment]
![Screenshot (104)](https://user-images.githubusercontent.com/68772150/154194907-6aa74686-d57e-4464-ae72-0bcfc9b3eb97.png)
This in Header tag

**Comment by caiyunlei** [comment]
I find two ways to import dashboard file by api.
https://blog.csdn.net/caiyunlei91/article/details/123353958?spm=1001.2014.3001.5501
